packages/jsx-email/test/config/load/dotdir/load-dotdir.test.ts

```diff
@@ -1,8 +1,20 @@
 import { loadConfig } from '../../../../src/config.js';
 
+const stable = (config: any) => {
```

Comment:
@CharlieHelps we have never needed this in any commit in the past. why have you added this now?

Reply:
@CharlieHelps that doesn't explain why the tests were passing without that helper, before your PR.